packages/components/src/lozenge/docs/Lozenge.stories.mdx

```diff
@@ -12,18 +12,35 @@ import { Text } from "@components/typography";
 
 # Lozenge
 
-<Tagline>Lozenges are used to highlight an item's status for quick recognition or to give additional fast short information such as highlighting a new item.</Tagline>
+<Tagline>Lozenges are used to highlight an item's status, for quick recognition or to give additional information such as highlighting a new item.</Tagline>
 
 <ComponentInfo
     usage={"import { Lozenge } from \"@sharegate/orbit-ui\";"}
     githubPath="/packages/components/src/lozenge/src"
 />
 
+## Best Practices
+
+- Use to highlight an element in a list.
+- Use to bring attention to a particular element that requires the user’s attention, such as an expiring license or a problematic situation.
+- Keep the copy short, ideally only 1 word or 2 short words. If you need to display more text, you should consider using another component such as a [message](?path=/docs/message--default-story) or an [alert](?path=/docs/alert--default-story).
+- Don't render this element interactive in any way, this includes making it clickable.
```

Comment:
One scenario that I could see happening a lot is using a `Tooltip` to add content to a Lozenge. Should we specify that it shouldn't be used with a Lozenge?

Reply:
> Don't render this element interactive in any way, this includes making it clickable

pretty much sums it up but adding that specification is not out of scope

Reply:
It's just that technically, adding a tooltip doesn't make it "clickable" per say. But are we ok with a tooltip on hover of a Lozenge?

Reply:
Don't render this element interactive was the hint / interactive includes hovering an element and reacting to it.